### PR TITLE
obs-ffmpeg: Add better error if srt or librist are not found

### PR DESCRIPTION
**Pull request by tytan652**
### Description

Just make the error if srt or librist are not found less cryptic.

### Motivation and Context

Avoid more issue like #6947 because of the cryptic error.

### How Has This Been Tested?

Didn't test it but it should work.

### Types of changes

- Tweak (non-breaking change to improve existing functionality)

### Checklist:

- [x] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [x] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [x] My code is not on the master branch.
- [ ] The code has been tested.
- [x] All commit messages are properly formatted and commits squashed where appropriate.
- [x] I have included updates to all appropriate documentation.

### CODE REVIEW

File: plugins/obs-ffmpeg/CMakeLists.txt [hunk 1]
```diff
@@ -18,8 +18,25 @@ add_library(OBS::ffmpeg ALIAS obs-ffmpeg)
...
+    )
+  elseif(NOT TARGET Libsrt::Libsrt)
+    obs_status(
+      FATAL_ERROR
+      "Srt library not found! Please install srt or set ENABLE_NEW_MPEGTS_OUTPUT=OFF."
+    )
+  elseif(NOT TARGET Librist::Librist)
+    obs_status(
+      FATAL_ERROR
+      "Librist library not found! Please install librist or set ENABLE_NEW_MPEGTS_OUTPUT=OFF."
```
Comment (RytoEX) [inline plugins/obs-ffmpeg/CMakeLists.txt:37]: Use SRT or libsrt and RIST or librist.

**Review by RytoEX — CHANGES_REQUESTED** [review]

**Comment by pkviet** [comment]
That's a good idea indeed.

File: plugins/obs-ffmpeg/CMakeLists.txt (hunk 1 above)
Comment (tytan652) [reply plugins/obs-ffmpeg/CMakeLists.txt:37]: `srt` seems to be commonly used as package name. So I guess `srt/libsrt` would may be the best solution.

Comment (RytoEX) [reply plugins/obs-ffmpeg/CMakeLists.txt]: SRT and RIST are acronyms and should be capitalized when used as such. The terms libsrt and libelist are not acronyms, so they do not require special capitalization. Furthermore, "SRT" and "RIST" indicating that they are acronyms helps provide a hint of what your desired search results should be, whereas "srt" and "rist" are not as clear. Please capitalize them the way that I have requested.

Comment (tytan652) [reply plugins/obs-ffmpeg/CMakeLists.txt]: Should be good now.

**Pull request merged**